the-collab-lab/website-next
# feature: render Stripe donation buttons at build time

## Summary

This PR refactors our approach to Stripe donations. Instead of getting data from the `products` collection, we get the `paymentLinks` collection, including the product data associated with that link. Having the payment links means that we don't need the client-side Stripe API at all.

## Test plan
1. Open the deploy preview
2. Scroll to the bottom of the homepage
3. Click on any of the Stripe donation links
4. Verify that you are taken to a Stripe checkout page
